.github/CODEOWNERS

```diff
@@ -1 +1 @@
-.github/CODEOWNERS @sonarsource/platform-onprem-squad
+.github/CODEOWNERS @Sonarsource/platform-onprem-squad
```

Comment:
If it is case sensitive I think it is supposed to be `SonarSource`.

Reply:
Arg yes indeed